README.md

```diff
@@ -135,87 +182,65 @@ OIDC tokens for Google Cloud Service Accounts are also accepted. These tokens in
 }
 ```
 
-The `repositories` and `permissions` attributes mirror the schema defined for [requesting a GitHub app installation access token](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app).
-
-* Repositories is an array of strings representing the name of the repository. Since the GitHub app only services a single organization per installation you do not need to specify the organization name as part of the repository name. E.g. `github-token-minter` instead of `abcxyz/github-token-minter`. The repository attribute supports a prefix wildcard match so you can select multiple repositories at once. A single `*` would capture all repositories for the organization but something like `github-*` would only capture repositories that started with `github-`.
+#### Scopes
 
-* Permissions is a map of permission name to access level. For example, to generate a token that can read issues you would specify `"issues": "read"`.
-
-### Deploying configuration to your service
+The scopes section contains named entry points into the configuration file. Each request contains a `scope` parameter which directs the service to look at the particular rules defined by that scope.
 
-The container built in this repository contains only the bare bones configuration required for the integration tests in our CI environment. All other repository configuration is stored outside of this repository and needs to be added into the container at deployment time.
+As you can see from the example, the `create-tag` scope defines another `if` statement which is used to verify that the caller is permitted to request this scope.
 
-We recommend that a GitHub workflow builds a new container from the released image produced by this repository that layers in your configuration files before it is deployed.
+The `repositories` and `permissions` attributes mirror the schema defined for [requesting a GitHub app installation access token](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app).
 
-Given a repository setup like
+* Repositories is an array of strings representing the name of the repository. The repository attribute supports a prefix wildcard match so you can select multiple repositories at once. A single `*` would capture all repositories for the organization but something like `github-*` would only capture repositories that started with `github-`. For single level repository configuration this field can be omitted and it will default to the current repository.
```

Comment:
didnt we say in repo config can only grant access to itself and not use this property?

Reply:
That is by convention and can't be enforced without some sort of policy solution.